data/scripts/sol-oic-gen.py

```diff
@@ -430,18 +430,18 @@ def generate_object_from_repr_vec_fn_common_c(name, props):
     const struct sol_oic_map_reader *repr_vec, uint32_t decode_mask)
 {
     struct sol_oic_repr_field field;
-    enum sol_oic_map_loop_status end_status;
```

Comment:
Reason outnumbers status because most of the places were not fixed after our API discussion. I don't have a strong opinion about that. I just think we need to choose one and fix it everywhere. I don't remember why we chose status. @cabelitos do you remember that?